include/fdeep/recurrent_ops.hpp

```diff
@@ -0,0 +1,184 @@
+// Copyright 2016, Tobias Hermann.
+// https://github.com/Dobiasd/frugally-deep
+// Distributed under the MIT License.
+// (See accompanying LICENSE file or at
+//  https://opensource.org/licenses/MIT)
+
+#pragma once
+
+#include <string>
+#include <functional>
+
+namespace fdeep { namespace internal
+{
+
+static float_type linear_activation(float_type x)
```

Comment:
As far as I know the `static` in front of free functions means two things:
- Semantics: It allows other translation units to have different functions with the same name.
- Performance optimizations: It allows inlining into other translation units.

Since frugally-deep is a header-only library, I think both points are irrelevant in our case, and I see no point of using `static` here. Please correct me if I'm missing something. Otherwise please remove the `static`. :)

Reply:
Yes you are right, I will remove `static`!